ir_webstats/client.py

```diff
@@ -200,6 +209,7 @@ def career_stats(self, custid=None):
         """ Gets career stats (top5, top 10, etc.) of driver (custid)."""
         r = self.__req(ct.URL_CAREER_STATS % (custid),
                        cookie=self.last_cookie)
+        #print(r)

```

Comment:
this should not be here

Reply:
Agreed. Forgive my newbieness, but how do I go about making that change into this pull request?

Reply:
You could just pull the code of the branch again. 
Though I doubt the original author will ever merge it. He hasn't responded to any of the pull requests.

https://github.com/a-tal/irace
This project seems more active :)